docs/core/experiment.rst

```diff
@@ -0,0 +1,24 @@
+Experiment
+==============================================================================
+
+Experiment - an abstraction that contains information about the experiment
+- a model, a criterion, an optimizer, a scheduler, and their hyperparameters.
+It also has information about the data and transformations used.
```

Comment:
`has information` -> `holds information`
`transformations used` -> `transformations to apply`